tests/ops/fixtures/saas_example_fixtures.py

```diff
@@ -309,7 +310,13 @@ def oauth2_authorization_code_connection_config(
 
 @pytest.fixture(scope="session")
 def saas_config() -> Dict[str, Any]:
-    saas_config: Dict[str, Any] = load_toml("saas_config.toml")
+    saas_config = {}
+    try:
+        saas_config = load_toml("saas_config.toml")
```

Comment:
Looks like we changed `load_toml` at some point to throw an exception if the file isn't found